packages/instantsearch.js/src/widgets/chat/chat.tsx

```diff
@@ -144,7 +145,7 @@ const createRenderer = <THit extends NonNullable<object> = BaseHit>({
   containerNode: HTMLElement;
   cssClasses: ChatCSSClasses;
   renderState: {
-    templateProps?: PreparedTemplateProps<Required<ChatTemplates<THit>>>;
+    templateProps?: PreparedTemplateProps<ChatTemplates<THit>>;
```

Comment:
this is normally set as required since everything has a default value. Is that no longer the case?